packages/tools/viewer/src/viewerElement.ts

```diff
@@ -683,6 +686,14 @@ export abstract class ViewerElement<ViewerClass extends Viewer = Viewer> extends
     })
     private _cameraTargetCoercer: Nullable<(camera: ArcRotateCamera) => void> = null;
 
+    /**
+     * The calculated hot spots for the scene cameras.
+     */
+    @state()
+    public camerasHotSpots: Record<string, HotSpot> = {};

```

Comment:
From my other comment, maybe this will be removed, but if we need to keep it, it seems like it should be private.